README.md

```diff
@@ -1,232 +1,110 @@
-[here](https://github.com/orgs/bisdn/packages?repo_name=docker-yocto-builder).
-Please make sure you pull the 'ci' version of the container linked above
-(or switch to the user 'builder' in the 'base' version).
-To continue with the steps documented in the section
-[Bootstrap build system](#bootstrap-build-system) you have to directly
-exec into the container, while ideally also mounting a local path on your
-file system (e.g. a new folder in your local /tmp) to the default TMPDIR
-directly inside of the container. To ensure that this /tmp directory will be
-writable by the builder user inside of the container, please make sure that you
-either have the same UID (1000) as the builder user inside of the container, or
-create the folder beforehand with more permissive access rights.
-This can be achieved by running:
+This repository holds files for building the
+[BISDN Linux](https://docs.bisdn.de/) network operating system.
 
-```bash
-# create a folder to be mounted as /tmp inside of the container with very
-# permissive access rights
-mkdir /tmp/builder-tmp && chmod 0777 /tmp/builder-tmp
-# start the docker container and mount the folder previously created as /tmp
-docker run -ti -v /tmp/builder-tmp:/tmp ghcr.io/bisdn/docker-yocto-builder-ci:main bash
-```
+Pre-built release images can be downloaded from the
+[BISDN Linux images](https://docs.bisdn.de/download_images.html) page.
+[Nightly builds](http://repo.bisdn.de/nightly_builds/) are also available.
 
-If you are planning to overwrite the TMPDIR in the conf/local.conf (`TMPDIR =
-"/tmp/${MACHINE}"` by default) as mentioned in the section
-[Configure target machine and cache directory](#configure-target-machine-and-cache-directory),
-you should also consider to mount your local file system path to this new path
-inside of your container.
+If you want to build your own BISDN Linux images, you can use the files
+in this repo and the kas automation tool. The only file that you will
+need is the kas configuration file `bisdn-linux.yaml`. All other files
+in this repo are documentation or for the benefit of our CI/CD and
+release management systems.
 
-After you have successfully exec'd into the container, you are ready to
-[bootstrap the build system](#bootstrap-build-system).
+The remainder of this document provides a quick guide on how to build
+a BISDN Linux image.
 
-## Installing your own build environment
+## Supported switches
 
-* repo tool
+A list of supported switches can be found at
+[BISDN Linux images](https://docs.bisdn.de/download_images.html).
 
-Some distros include [repo](https://android.googlesource.com/tools/repo), so
-you might be able to install from there (Ubuntu 20.04 however does not provide
-those packages).
+## Build requirements
 
-```bash
-# Debian/Ubuntu.
-sudo apt-get install repo
-```
+### Hardware requirements
 
-```bash
-# Gentoo.
-sudo emerge dev-vcs/repo
-```
+We recommend a build host with 16 CPU cores, 32 GiB RAM and 150 GiB free
```

Comment:
```suggestion
We recommend a build host with 32 CPU cores, 32 GiB RAM and 150 GiB free
```

Reply:
to match the time and disk space section below

Reply:
Done.